networking/dns-operator.adoc

```diff
@@ -0,0 +1,8 @@
+[id="dns-operator"]
+= Understanding the Operator in {product-title}
```

Comment:
Should this be "Understanding the DNS Operator"?

Reply:
Yep :man_facepalming: